app/controllers/books_controller.rb

```diff
@@ -2,4 +2,17 @@ class BooksController < ApplicationController
   def index
     @books = Book.all
   end
+  def create
+    @book = Book.new(book_params)
+     if @book.save
+        flash[:success] = "New Book created."
+        redirect_to '/books/index'
```

Comment:
Just a heads up: Rails automatically generates path helpers so you can just do `redirect_to books_path`